src/inc/gcdecoder.cpp

```diff
@@ -293,162 +316,164 @@ const InfoHdrSmall infoHdrShortcut[128] = {
-       { 13,  8, 1, 0, 1, 1, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0,  0,  0,  9,  0, YES         },  //     627  6b
-       { 13,  8, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,  0,  0,  2,  1,  0          },  //    1099  6c
-       { 13, 10, 1, 0, 1, 1, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0,  0,  2,  6,  1, YES         },  //     488  6d
-       { 14,  7, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0,  0,  0,  1,  0, YES         },  //     574  6e
-       { 16,  7, 1, 0, 1, 1, 1, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0,  0,  0,  4,  0, YES         },  //    1281  6f
-       { 16,  7, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0,  0,  0,  0,  0, YES         },  //    1881  70
-       { 16,  7, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0,  0,  0,  1,  0, YES         },  //     339  71
-       { 16,  7, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0,  0,  0,  3,  0,  0          },  //    2594  72
-       { 16,  7, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0,  0,  0,  4,  0,  0          },  //     339  73
-       { 16,  7, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0,  0,  0,  4,  0, YES         },  //    2107  74
-       { 16,  7, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0,  0,  0,  5,  0, YES         },  //    2372  75
-       { 16,  7, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0,  0,  0,  6,  0, YES         },  //    1078  76
-       { 16,  7, 2, 0, 1, 1, 1, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0,  0,  0,  4,  0, YES         },  //     384  77
-       { 16,  9, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0,  0,  1,  4,  1, YES         },  //    1541  78
-       { 16,  9, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0,  0,  2,  4,  1, YES         },  //     975  79
-       { 19,  7, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 1, 0, 0, 0, 0, 0,  0,  0,  5,  0, YES         },  //     546  7a
-       { 24,  7, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0,  0,  0,  5,  0, YES         },  //     675  7b
-       { 45,  9, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0,  0,  1,  0,  0,  0          },  //     902  7c
-       { 51,  7, 1, 0, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0,  0,  0, 13,  0, YES         },  //     432  7d
-       { 51,  7, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0,  0,  0,  1,  0, YES         },  //     361  7e
-       { 51,  7, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0,  0,  0, 11,  0,  0          },  //     703  7f
+//        |   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  genericsContextIsMethodDesc
+//        |   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
+//        |   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  returnKind
+//        |   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
+//        |   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  Arg count
+//        |   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |                                 Counted occurances
```

Comment:
Since you are editing this file, could you please fix the spelling of the comment here: occurances -> occurences